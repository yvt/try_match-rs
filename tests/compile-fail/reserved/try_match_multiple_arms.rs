fn main() {
    let _ = try_match::try_match!(Some(42), Some(_) => 1, None => 0);
}
