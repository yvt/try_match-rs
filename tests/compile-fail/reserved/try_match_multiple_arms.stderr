error: no rules expected the token `,`
 --> tests/compile-fail/reserved/try_match_multiple_arms.rs:2:57
  |
2 |     let _ = try_match::try_match!(Some(42), Some(_) => 1, None => 0);
  |                                                         ^ no rules expected this token in macro call
  |
note: while trying to match meta-variable `$out:expr`
 --> src/lib.rs
  |
  |     ($in:expr, $(|)? $($p:pat)|+ $(if $guard:expr)? => $out:expr) => {
  |                                                        ^^^^^^^^^
